py/server/setup.py

```diff
@@ -52,8 +52,8 @@ def normalize_version(version):
     keywords='Deephaven Development',
     python_requires='>=3.7',
     install_requires=[
-        'jpy>=0.11.0',
-        'deephaven-plugin>=0.3.0.dev2',
+        'jpy>=0.12.0',
+        'deephaven-plugin>=0.3.0.dev3',
```

Comment:
Don't think we should reference a dev2 version

Reply:
Yep - this will be updated once the deephaven-plugin side is merged an release (before this PR merges).